docs/src/app/components/pages/components/SelectField/ExampleNullable.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+const styles = {
+  customWidth: {
+    width: 150,
+  },
+};
+
+/**
+ * `SelectField` can also be nullable. In this case, just specify a `MenuItem`
+ * with no text and with a `null` value. For instance, for a boolean:
+ */
+export default class SelectFieldExampleNullable extends React.Component {
+
+  constructor(props) {
```

Comment:
This block could be replaced by:
```jsx
state = {
  value: null,
};
```
I would rather avoid calling the constructor.